doc/manuals/admin/logs.md

```diff
@@ -108,52 +110,76 @@ The different fields in each line are as follows:
     include the name of the file and line number generating the trace
     (this information is useful mainly for Orion developers).
 
-[Top](#top)
-    
-## Error and warning types
+[Top](#top)  
+
+## Alarms
+
+Alarm conditions:
+
+| Alarm ID   | Severity   |   Detection strategy                                                                                              | Stop condition                                                                                                                                                                                                                            | Description                                                                                                  | Action
+|:---------- |:----------:|:----------------------------------------------------------------------------------------------------------------- |:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |:------------------------------------------------------------------------------------------------------------ |:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+| 1          | CRITICAL   | A FATAL trace is found                                                                                            | N/A                                                                                                                                                                                                                                       | A problem has occurred at Orion Context Broker startup. The FATAL 'msg' field details the particular problem. | Solving the issue that is precluding Orion Context Broker startup, e.g. if the problem was due to the listening port is being used, the solution would be either changing Orion listening port or ending the process that is already using the port.
+| 2          | CRITICAL   | The following ERROR text appears in the 'msg' field: "Runtime Error (`<detail>`)"                                 | N/A                                                                                                                                                                                                                                       | Runtime Error. The `<detail>` text containts the detailed information.          | Restart Orion Context Broker. If it persists (e.g. new Runtime Errors appear within the next hour), scale up the problem to development team.
```

Comment:
Not sure about "CRITICAL" for "Runtime Error". Currently, they correspond to things that it shouldn't happend but Orion uses to be quite protected agains crash due to them. However, they are still critical in the sense that developers team should be reported about that in order to have a look...